PhoneFormat.js

```diff
@@ -16,6 +11,34 @@ Phoneformat._getCountryForIp = function (callback) {
   });
 };
 
+// Get the user's current country
+Phoneformat._getCountryForIp = function (callback) {
+  // Return cached country if it exists.
+  var storedCountry = localStorage.getItem('phoneformat.current_country');
+  if (storedCountry) return callback && callback(storedCountry);
+
+  // If the user is on a mobile device, attempt to get the country from the phone.
+  if (Meteor.isCordova) {
```

Comment:
You should check to see if globalization is available -- just in case the plugin fails.